tests/lib/rules/use-isnan.js

```diff
@@ -280,16 +280,6 @@ ruleTester.run("use-isnan", rule, {
             code: "switch(foo) { case NaN: break; }",
             errors: [{ messageId: "caseNaN", type: "SwitchCase", column: 15 }]
         },
-        {
```

Comment:
Why were these tests removed?

Reply:
this was removed because i removed the `[{}]` option and it was duplicate case to when I ran the linter, it removed them. 

I am fixing this now 👍